.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Not only `node_modules` but also `build` and `coverage` should be omitted. All we need to preserve is source code.

Reply:
If you're interested in showing `coverage` in PR, you can try to use [codecov](https://github.com/apps/codecov) in future PR.